lighthouse-core/gather/driver.js

```diff
@@ -630,6 +630,31 @@ class Driver {
     return new LHElement(targetNode, this);
   }
 
+  /**
+   * @param {string} handle
+   * @param {{timeout: number}=} options,
+   * @return {Promise<string>}
+   */
+  async readIOStream(handle, options = {timeout: 5000}) {
```

Comment:
nit: this might be better in `fetcher.js`. It is a general `readIOStream` utility method, but the only `IO.StreamHandle` we'd ever be reading is this one (the only other relevant one is traceEvents and we moved off that a long time ago). It's ok if you'd rather it be here, though.